api/tests/routes/backoffice_v3/finance_test.py

```diff
@@ -78,6 +78,24 @@ def test_get_incident_cancellation_form(self, legit_user, authenticated_client):
         assert response.status_code == 200
 
 
+class GetIncidentValidationFormTest(GetEndpointHelper):
+    endpoint = "backoffice_v3_web.finance_incident.get_finance_incident_validation_form"
+    endpoint_kwargs = {"finance_incident_id": 1}
+    needed_permission = perm_models.Permissions.MANAGE_INCIDENTS
+
+    expected_num_queries = 2
+    expected_num_queries += 3  # get incident info
```

Comment:
Avec les bons joinedload, on devrait pouvoir redescendre :)
```suggestion
    expected_num_queries += 1  # get incident info
```